ci/builders/windows_arm_host_engine.json

```diff
@@ -0,0 +1,124 @@
+{
+    "builds": [
+        {
+            "archives": [
+                {
+                    "base_path": "out/host_debug_arm64/zip_archives/",
+                    "type": "gcs",
+                    "include_paths": [
+                        "out/host_debug_arm64/zip_archives/windows-arm64/artifacts.zip",
+                        "out/host_debug_arm64/zip_archives/windows-arm64/windows-arm64-embedder.zip",
+                        "out/host_debug_arm64/zip_archives/windows-arm64/font-subset.zip",
+                        "out/host_debug_arm64/zip_archives/dart-sdk-windows-arm64.zip",
+                        "out/host_debug_arm64/zip_archives/windows-arm64-debug/windows-arm64-flutter.zip",
+                        "out/host_debug_arm64/zip_archives/windows-arm64/flutter-cpp-client-wrapper.zip",
+                        "out/host_debug_arm64/zip_archives/flutter-web-sdk-windows-arm64.zip"
```

Comment:
I got the following archives instead:

1. `out/host_debug_arm64/zip_archives/windows-x64/windows-x64-embedder.zip`
2. `out/host_debug_arm64/zip_archives/windows-x64/flutter-cpp-client-wrapper.zip`

Here are the commands I used to build:

```
.\tools\gn --runtime-mode debug --full-dart-sdk --no-lto --no-goma --windows-cpu arm64
```

```
autoninja -C ..\out\host_debug_arm64 flutter/build/archives:artifacts flutter/build/archives:embedder flutter/tools/font-subset flutter/build/archives:dart_sdk_archive flutter/shell/platform/windows/client_wrapper:client_wrapper_archive flutter/build/archives:windows_flutter flutter/web_sdk
```

Reply:
It looks like you'll need to update these lines as well:

https://github.com/flutter/engine/blob/1d2ba73d1059544564e644055d4366956e581e87/shell/platform/windows/client_wrapper/BUILD.gn#L116

https://github.com/flutter/engine/blob/1d2ba73d1059544564e644055d4366956e581e87/shell/platform/embedder/BUILD.gn#L533

Reply:
Done